src/karma-test-shim.ts

```diff
@@ -3,6 +3,7 @@ Error.stackTraceLimit = Infinity;
 // import the stylesheet
 import '../node_modules/bootstrap/dist/css/bootstrap.css';
 import './styles/ux-aspects.less';
+import './styles/hpe-icons.css';
```

Comment:
Is this needed? Do any of our components still use this?

Reply:
The reorderable-table component uses hpe-icons. @donna-marijne Do we want to change this to be ux-aspects as part of this ticket?

Reply:
Yes, you can change that now if you wish, or create a ticket to have it fixed.

Reply:
Ok sure. I'll do it as part of this ticket.

Reply:
Removed now.